src/node_api_jsrt.cc

```diff
@@ -348,16 +348,17 @@ const char* error_messages[] = {
   "Unknown failure",
   "An exception is pending",
   "The async work item was cancelled",
+  "napi_escape_handle already called on scope"
 };
 
 napi_status napi_get_last_error_info(napi_env env,
                                      const napi_extended_error_info** result) {
   CHECK_ARG(result);
 
   static_assert(
-    sizeof(error_messages) / sizeof(*error_messages) == napi_status_last,
+    node::arraysize(error_messages) == napi_escape_called_twice + 1,
```

Comment:
Yeah, that PR was closed now and we're going a different direction.  LGTM as is.